src/app/systelab-components/datepicker/datepicker.component.i18n.spec.ts

```diff
@@ -23,12 +23,16 @@ export class USMockI18nService {
 	}
 
 	public getDateFormatForDatePicker() {
-		return 'mm/dd/yy';
+		return 'MM/DD/YY';
```

Comment:
This mock is not returning the same that original service

- systelab-translate/i18n.service.ts/getDateFormatForDatePicker   
-->  calls to --> systelab-translate/ date-util.ts/getDateFormatForDatePicker   

https://github.com/systelab/systelab-translate/blob/master/src/app/systelab-translate/date-util/date-util.ts

```
	public getDateFormatForDatePicker(isFullYear = false): string {
		let stringDateFormat = '';
		switch (this.locale) {
			case 'en-US':
				stringDateFormat = 'm/d/y';
				break;
...
		if (isFullYear) {
			stringDateFormat = stringDateFormat.replace('y', 'yy');
			if (this.locale === 'en-US') {
				stringDateFormat = stringDateFormat.replace('m', 'mm');
				stringDateFormat = stringDateFormat.replace('d', 'dd');
			}
		}
		return stringDateFormat;
```

Reply:
You're right!